bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/InternalHooks.java

```diff
@@ -0,0 +1,17 @@
+package com.bugsnag.android;
+
+import kotlin.Unit;
+import kotlin.jvm.functions.Function0;
+
+class InternalHooks {
+
+    private final Client client;
+
+    public InternalHooks(Client client) {
+        this.client = client;
+    }
+
+    public void setEventStoreEmptyCallback(Function0<Unit> callback) {
```

Comment:
I don't think we need to instantiate this class

```suggestion
    private InternalHooks() {}

    public static void setEventStoreEmptyCallback(Client client, Function0<Unit> callback) {
```